rwcore/loaders/LoaderIMG.hpp

```diff
@@ -36,7 +36,7 @@ class LoaderIMG {
     /// Load the structure of the archive
     /// Omit the extension in filename so both .dir and .img are loaded when
     /// appropriate
-    bool load(const rwfs::path& filename);
+    bool load(const rwfs::path& filepath);
```

Comment:
nit: camelCase (as you already have changed this :P )

Reply:
camelCased it for you